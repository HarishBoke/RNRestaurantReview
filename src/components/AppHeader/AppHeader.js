import React from 'react';
import {View, Text} from 'react-native';
import AppHeaderStyle from './AppHeader.style';

const AppHeader = () => (
  <View style={AppHeaderStyle.pageTitle}>
    <Text style={AppHeaderStyle.pageTitleText}>Restaurant Review</Text>
  </View>
);
export default AppHeader;
