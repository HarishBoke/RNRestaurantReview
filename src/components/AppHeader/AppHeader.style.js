import React from 'react';
import StyleSheet from 'react-native';

export default {
  pageTitle: {
    padding: 40,
  },
  pageTitleText: {
    fontSize: 24,
    fontWeight: '300',
    color: 'blue',
    textAlign: 'center',
  },
  iOSPageTitleText: {
    fontSize: 24,
    fontWeight: '300',
    color: 'green',
    textAlign: 'center',
  },
};
