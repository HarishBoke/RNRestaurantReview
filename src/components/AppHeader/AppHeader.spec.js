import React from 'react';

import {shallow} from 'enzyme';

import AppHeader from './AppHeader';

const initWrapper = (overrides) => {
  const mockMethods = {};

  const mockProps = {};

  const wrapper = shallow(
    <AppHeader {...mockMethods} {...mockProps} {...overrides} />,
  );

  return {mockMethods, mockProps, wrapper};
};

describe('AppHeader', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const component = shallow(<AppHeader label="test label" />);
      expect(component).toMatchSnapshot();
    });
  });
});
