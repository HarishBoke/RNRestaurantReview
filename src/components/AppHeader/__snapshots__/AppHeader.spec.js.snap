// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
